app/enterprise/1.3-x/developer-portal/structure-and-file-types.md

```diff
@@ -166,10 +186,10 @@ x-headmatter:
 ...
 ```
 
-Specs are a collection meaning their `layout` and `route` are determined by portal configuration and not the file itself.  Specs are rendered with the `system/spec-renderer.html` layout, under the route pattern `/documentation/:name` where name is the name of the particular spec file.  So a spec with a path of `specs/myfirstspec.json` will render in the portal as `/documentation/myfirstspec`.  If you would like to overwrite the hard coded spec collection config, you can do so by including your own in `portal.conf.yaml`.
-
+Specs are a collection meaning their `layout` and `route` are determined by portal configuration and not the file itself.  Specs are rendered by default by the `system/spec-renderer.html` layout, under the route pattern `/documentation/:name` where name is the name of the particular spec file.  So a spec with a path of `specs/myfirstspec.json` will render in the portal as `/documentation/myfirstspec`.  If you would like to overwrite the hard coded spec collection config, you can do so by including your own in `portal.conf.yaml`.  Check out the Collections section of our `Working with Templates` guide to learn more.
```

Comment:
```suggestion
Specs are a collection meaning their `layout` and `route` are determined by the portal configuration and not the file itself.  Specs are rendered by default by the `system/spec-renderer.html` layout, under the route pattern `/documentation/:name` where name is the name of the particular spec file.  So a spec with a path of `specs/myfirstspec.json` will render in the portal as `/documentation/myfirstspec`.  If you would like to overwrite the hardcoded spec collection config, you can do so by including your own in `portal.conf.yaml`.  Check out the Collections section of our `Working with Templates` guide to learn more.
```

Reply:
Not sure why it doesn't highlight diffs, but there were some missing articles and punctuation.